### ISSUE TITLE
Bundle

### ISSUE THREAD

**Issue by vCrespoP** [KrauseFx/TSMessages#213]

**Comment by jondot** [comment]
The error is 
```
*** Assertion failure in +[TSMessageView notificationDesign], /Users/.../TSMessages/Pod/Classes/TSMessageView.m:68
```

**Comment by nunogoncalves** [comment]
+1.

**Comment by mRs-** [comment]
Hand in a Pull request :)

**Comment by nunogoncalves** [comment]
I might be missing something here, but pointing the pod to github, I don't have this issue. If I set the cocoapods version it does have this problem.

**Comment by kiliankoe** [comment]
Looks like this was fixed in 38d3d83c62f176b7b72e9f3aa24a3aaa729a7857 which is after the latest release 0.9.12.

@KrauseFx It'd be awesome if you could release a new version with this fix so that it can be installed via cocoapods :grin: Thanks!

**Comment by davidrothera** [comment]
Yeah, @KrauseFx can we get a version bump on this one? ;)

**Comment by KrauseFx** [comment]
@kiliankoe @davidrothera To be honest I'm afraid a new version introduces new errors, as the last release has been ages ago... I haven't had time to maintain TSMessages recently unfortunately. I'd like to bring TSMessages up to date again soon.

Is there a reason you don't want to define the `:head` option in your `Podfile`?

**Comment by davidrothera** [comment]
Yeah, I've gone that route for now, its only for a development branch at the moment so no real danger using :head and not a tagged version :+1:

**Comment by adamsp** [comment]
+1 on a CocoaPods release for this.

**Comment by fjsosa** [comment]
+1 on a CocoaPods release for this, Head version uses an old version of HexColors (2.2.0). Thank you in advance.

**Comment by BenjaminDigeon** [comment]
:+1:  on a CocoaPods release for this too !

**Comment by IceFloe** [comment]
+1 on a Cocoapods
